src/main/java/com/google/cloud/tools/managedcloudsdk/ManagedCloudSdk.java

```diff
@@ -166,8 +168,13 @@ public boolean isUpToDate() throws ManagedSdkVerificationException {
 
   // TODO : fix passthrough for useragent and client side usage reporting
   public SdkInstaller newInstaller() {
-    String userAgentString = "google-cloud-tools-java";
-    return SdkInstaller.newInstaller(managedSdkDirectory, version, osInfo, userAgentString, false);
+    return SdkInstaller.newInstaller(
+        managedSdkDirectory, version, osInfo, USER_AGENT_STRING, false, Collections.emptyMap());
```

Comment:
could you push the user agent string into SdkInstaller.newInstaller? Or does something set a different value?

Reply:
I would leave that for @chanseokoh since that is not related to this change. Chanseok, could you confirm this is always hardcoded and if so, should we file an issue for that?

Reply:
I have no idea about this. But I see a TODO on the previous line, so maybe the current signature is intended.

```
  // TODO : fix passthrough for useragent and client side usage reporting
```